src/fontra/views/fontoverview/fontoverview.js

```diff
@@ -340,6 +340,19 @@ export class FontOverviewController extends ViewController {
     const glyphItemList = this.glyphOrganizer.filterGlyphs(combinedGlyphItemList);
     const glyphSections = this.glyphOrganizer.groupGlyphs(glyphItemList);
     this.glyphCellView.setGlyphSections(glyphSections);
+
+    // Add placeholder if no glyphs are found
+    if (glyphSections.length === 0) {
```

Comment:
Please look at the related glyphs panel like I told you. There is a callback for this.

Reply:
I assume it would look like this:
<img width="1618" alt="Screenshot 2025-01-15 at 21 40 20" src="https://github.com/user-attachments/assets/72565210-6f58-4bb6-93c2-a220a8724e28" />

But this does not work. 
Within `setGlyphSections` it does not get to `this.onNoGlyphsToDisplay?.();` because `this.accordion.items.length` is 0, this is not the case for 'related glyphs'. So, there must be a difference between font-overview and related glyphs which I don't understand, currently.

Reply:
When the search field in the font-overview is empty -> `this.accordion.items` is also empty. Which makes sense to me. This is different for 'related glyphs' it still has 5 items in `this.accordion.items`, even though it displays '(No related glyphs or characters were found)'.

Reply:
Ahhh yes, so we know for sure there are glyphs once there are accordion items, and no glyphs if there are no accorion items. And we don't have the situation where sections can turn out to be empty after the fact, as in the related glyph panel. So this situation is simpler, and your check for "are there items" is always enough. Apologies.

Reply:
Thanks. 
I am not sure what to do now. Do we keep the PR as it is or do we modify other parts of the code?

Reply:
The logic is correct, right? I have some feedback about the details of the placeholder element, but otherwise this is good.